src/app/study/containers/study-page.component.html

```diff
@@ -37,25 +37,41 @@
   <ng-template #loadedAndValid>
 
     <app-tile label="Subjects">
+      <div header-content>
+        <button
+          mat-icon-button
+          (click)="refreshSubjectTable()"
+        >
+          <i class="material-icons">refresh</i>
+        </button>
+      </div>
+
       <div
         tile-content
         class="tile-content-subjects"
       >
-        <app-subject-table
-          [subjects]="subjects$ | async"
-          [studyName]="(study$ | async)?.projectName"
-          [isLoaded]="subjectsLoaded$ | async"
-          (openSubject)="openSubjectHandler($event)"
-        ></app-subject-table>
-        <app-subject-detail
-          *ngIf="subject"
-          [subject]="subject"
-        ></app-subject-detail>
+        <section
+          [class.subject-table]="subject"
+          [class.subject-table-before]="!subject"
```

Comment:
I find this [syntax](https://toddmotto.com/ng-class-angular-classes#multiple-classes-with-ngclass) easy to read.